src/components/GGanttColumn.vue

```diff
@@ -0,0 +1,70 @@
+<template>
+  <div class="g-gantt-column" :style="{ fontFamily: font, color: colors.text }">
+    <span class="g-gantt-header" :style="{ background: colors.primary }">{{ columnTitle }}</span>
```

Comment:
While we are at it, we could expose a `<slot>`for the label column title.

Reply:
by default, column title and label are the same behaviour before, but now it's possible to change them with a `<slot>` tag

Default behaviour:
![image](https://github.com/zunnzunn/vue-ganttastic/assets/51349076/27d8cab9-5444-43e3-a1b1-5eba1e531481)


Custom behaviour:
![image](https://github.com/zunnzunn/vue-ganttastic/assets/51349076/b4741df3-1476-4010-a5d8-2c0eda0d65ce)
![image](https://github.com/zunnzunn/vue-ganttastic/assets/51349076/2b2e3a9b-8a91-45ff-baa0-4ffa9a615133)